be/src/service/starrocks_main.cpp

```diff
@@ -111,7 +111,12 @@ int main(int argc, char** argv) {
     using std::string;
 
     // Open pid file, obtain file lock and save pid.
-    string pid_file = string(getenv("PID_DIR")) + "/be.pid";
+    string pid_file = string(getenv("PID_DIR"));
+    if (as_cn) {
+        pid_file += "/cn.pid";
```

Comment:
Could we keep using `be.pid` and `be.conf`, and not introducing new file like `cn.pid` and `cn.conf`.